src/LicenseHeader.cs

```diff
@@ -0,0 +1,30 @@
+using AutoRest.TypeScript.DSL;
```

Comment:
Would you mind creating an `utilities` Folder and moving this file there?

Reply:
Sure